openassessment/locale/config.yaml

```diff
@@ -74,11 +74,19 @@ dummy_locales:
 
 # Directories we don't search for strings.
 ignore_dirs:
```

Comment:
I don't think this configuration does anything if you're not using `i18n_tool extract`. If you're using just makemessages you need to explicitly pass `--ignore` option. https://docs.djangoproject.com/en/1.4/ref/django-admin/#django-admin-option---ignore